app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ]  Please the post controller should contain a new and a create method that handles post Post request, please add it to your controller 
![image](https://user-images.githubusercontent.com/74744787/222924525-68e4e4e0-4779-4ad1-8a74-d1bd39968e58.png)

Reply:
Hi @MeqdamAlqudah,

Thank you for your review. I have resolved these.